packages/core/stories/card/card.stories.tsx

```diff
@@ -334,10 +334,60 @@ export const InteractableAsBlockLinkWithImageBackground: StoryFn<
         <div
           style={{ paddingTop: "calc(3 * var(--salt-size-container-spacing))" }}
         >
-          <H1 styleAs="h3">{exampleData[0].title}</H1>
+          <H3>{exampleData[0].title}</H3>
           <Text>{exampleData[0].content}</Text>
         </div>
       </InteractableCard>
     </Link>
   );
 };
+
+export const CardAndInteractableCardSizes: StoryFn<typeof Card> = () => {
+  const sizes = ["small", "medium", "large"];
+  return (
+    <StackLayout style={{ width: 600 }}>
+      {sizes.map((size, index) => {
+        return (
+          <StackLayout direction="row">
```

Comment:
The key should be here instead of on InteractableCard and we should avoid using indexes as keys and use the size instead.